### ISSUE TITLE
[Bug] Character orientation and pose locked, independent of camera movement

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2127]
I don't remember exactly how I did it but here are some clues.

When messing around with the Function row keys (F1-F8) and teleportation in the Codex, suddenly I couldn't rotate my character's model. I could move the camera but the model was stuck in the same orientation as before as present on this screenshot:
![image](https://user-images.githubusercontent.com/63756057/82673516-e78bf700-9c39-11ea-9381-eba0c8fcbabd.png)
The version of the game is current (20w21c) but now it shows "pre-alpha caca" for some reason.
Here is the savefile so you can experiment yourself:
[Indrasil.zip](https://github.com/tsunamayo/Starship-EVO/files/4668382/Indrasil.zip)
The log was 27 MB for some reason but GitHub accepts files smaller than 10 MB only, so here's the link: https://1drv.ms/t/s!AmcHNSRjXywp1AYgIgdNdpU3wQJ1?e=VnEAvy

**Comment by Aiwii** [comment]
I think F6 or F10 is dev cam

**Comment by Kaiser-Indrasil** [comment]
> 
> 
> I think F6 or F10 is dev cam

Nah, I already tried it. Doesn't work.

**Comment by tsunamayo** [comment]
There is a lot of issue in that log file...
I am loading your save without an issue. Can you reload your game too?
Did you touch an hotkey at one moment? Also did you changed your skin? Thanks

**Comment by Kaiser-Indrasil** [comment]
I probably did change my skin.
Sorry, but the save is long gone.

**Comment by ShannonW1950** [comment]
I was asked to put my situation in here, it's similar but not quite the same.

The game had been running with no issues for a while now, but then today I've started having an issue since coming back from dinner. The starting point I believe was exiting to menu. When I loaded my character the first time, I was stuck in third-person, stuck (I think) inside my build, my toolgun was floating next to me, no keys or mouse control worked but there was still grass and player idle animation so I'm assuming it wasn't frozen. Version was 'pre-build caca'.

After ending task, I tried again. This time, I loaded in the same location, still stuck but this time no character model. However I could access the escape key menu and some function keys appeared to work. Still no mouse control, movement, codex etc. Version shows as 'pre-build 20w21c'

The output_log is 39 mb so I'm linking it. https://u.pcloud.link/publink/show?code=XZ17TqkZQtu7oodfFljKUjqkki0OO0LWxYcy
